opds_import.py

```diff
@@ -353,7 +353,19 @@ class OPDSImporter(object):
         {
             "key": Collection.DATA_SOURCE_NAME_SETTING,
             "label": _("Data source name"),
-        }
+        },
+        {
+            "key": ExternalIntegration.USERNAME,
+            "label": _("Username"),
+            "optional": True,
+            "description": _("If authentication is required to access the OPDS feed, enter the username here."),
+        },
+        {
+            "key": ExternalIntegration.PASSWORD,
+            "label": _("Password"),
+            "optional": True,
+            "description": _("If authentication is required to access the OPDS feed, enter the password here."),
```

Comment:
I like that this is part of the OPDS import rather than a whole new protocol, but I think the description needs something to distinguish between this case and the OPDS For Distributors case. Something like this:

"If HTTP Basic authentication is required to access the OPDS feed (it usually isn't)..."